angular/projects/catalyst/src/lib/formly/formly.module.ts

```diff
@@ -0,0 +1,78 @@
+import { NgModule } from '@angular/core';
+import { FormlyModule } from '@ngx-formly/core';
+import { CatCheckboxFieldType } from './formly-checkbox/formly-checkbox.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { CatalystModule } from '../catalyst.module';
+import { CatToggleFieldType } from './formly-toggle/formly-toggle.component';
+import { CatRadioFieldType } from './formly-radio/formly-radio.component';
+import { CatInputFieldType } from './formly-input/formly-input.component';
+import { CatTextareaFieldType } from './formly-textarea/formly-textarea.component';
+import { CatRadioGroupFieldType } from './formly-radio-group/formly-radio-group.component';
+import { CommonModule } from '@angular/common';
+import { CatDatepickerFieldType } from './formly-datepicker/formly-datepicker.component';
+import { CatSelectFieldType } from './formly-select/formly-select.component';
+
+@NgModule({
+  declarations: [
+    CatCheckboxFieldType,
+    CatToggleFieldType,
+    CatRadioFieldType,
+    CatRadioGroupFieldType,
+    CatInputFieldType,
+    CatTextareaFieldType,
+    CatDatepickerFieldType,
+    CatSelectFieldType
+  ],
+  imports: [
+    CatalystModule,
+    ReactiveFormsModule,
+    CommonModule,
+    FormlyModule.forChild({
+      types: [
+        {
+          name: 'cat-checkbox-formly',
+          component: CatCheckboxFieldType,
+        },
+        {
+          name: 'cat-toggle-formly',
+          component: CatToggleFieldType,
+        },
+        {
+          name: 'cat-radio-formly',
+          component: CatRadioFieldType,
+        },
+        {
+          name: 'cat-radio-group-formly',
+          component: CatRadioGroupFieldType,
+        },
+        {
+          name: 'cat-input-formly',
+          component: CatInputFieldType,
+        },
+        {
+          name: 'cat-textarea-formly',
+          component: CatTextareaFieldType,
+        },
+        {
+          name: 'cat-datepicker-formly',
+          component: CatDatepickerFieldType,
+        },
+        {
+          name: 'cat-select-formly',
+          component: CatSelectFieldType,
+        }
+      ],
+    }),
+  ],
+  exports: [
+    CatCheckboxFieldType,
+    CatToggleFieldType,
+    CatRadioFieldType,
+    CatRadioGroupFieldType,
+    CatInputFieldType,
+    CatTextareaFieldType,
+    CatDatepickerFieldType,
+    CatSelectFieldType
+  ],
+})
+export class CatalystFormlyModule {}
```

Comment:
Why are we not importing this anywhere in the demo app? Shouldn't we have to?

Reply:
I've exported also each component separately in the public-api file.. Probably thats why the components are imported directly from the catalyst library. But if you want to use the custom aliases you should import the CatalystFormlyModule
![image](https://github.com/haiilo/catalyst/assets/89849556/105e61c4-a786-418f-92dd-b36bbaea8f1d)